<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chat_ui</class>
 <widget class="QMainWindow" name="chat_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>当前在线用户</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="chat_show">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="user_name">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QComboBox" name="feature">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>180</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>挂机</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>私聊</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>广播</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>禁言（管理员权限）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>解禁（管理员权限）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>踢人（管理员权限）</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>请选择功能：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>210</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>请选择对象</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ob">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>230</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputtext">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>啊哈聊天室</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
